### ISSUE TITLE
onChange and value

### ISSUE THREAD

**Issue by Nondv** [tonsky/rum#152]
Hello!
I assume that if `:value` attribute is constant it will be always the same.
But I can easily change value of this component:

```
(rum/defc  my-input []
  [:input {:on-change js/console.log :value "whatever"}])
```

But! if I completely remove `:on-change` it will be all right.
Is it expected behavior? What should I do?

**Comment by Nondv** [comment]
Well, it appears that Sablono wraps elements with value (such as select and input):
it creates component around element and creates element state.

https://github.com/tonsky/sablono/blob/master/src/sablono/interpreter.cljc#L27

In short, it does not produce this:

```html
<input value="whatever" onChange={log} />
```

but this:

```html
<wrapper value="whatever" onChange={log}>
  <input value={inputState.state_value} onChange={innerOnChange} />
</wrapper>
```

where `innerOnChange` is like

```javascript
function(v) {
  if(!wrapper.props.onChange) return;

  input.setState({ state_value: v })
  wrapper.props.onChange(v);
}
```

**Comment by Nondv** [comment]
https://github.com/r0man/sablono/issues/148

**Comment by tonsky** [comment]
Yeah. Inputs are a huge mess in Rum/Sablono/React integration. I hope to fix or at least clean it up one day

**Comment by rauhs** [comment]
I think the simplest would be to throw out the wrapped components and *synchronously* call `inst.forceUpdate()` on a change to `:rum/local`. Then we'd just have the same behaviour as the React folks.

**Comment by tonsky** [comment]
that is true. Have to ditch sablono first though

**Comment by roman01la** [comment]
@Nondv Please try to repro this in `0.12.0-SNAPSHOT` and report any issues to https://github.com/tonsky/rum/pull/205

**Comment by roman01la** [comment]
fixed in 0.12.0
